challenge.rb

```diff
@@ -44,8 +54,18 @@ def okay_to_drink? age, drinking_age
 # If the AC is non-functional and it's NOT too hot, `ac_monitor` should return
 #   "Fix the AC whenever you have the chance. It's cool."
 
+# did't want to do nested ifs
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ac_working == true && current_temp > desired_temp
+    status = "Turn on the AC."
+  elsif ac_working == true && current_temp < desired_temp
+    status = "Just right!"
+  elsif ac_working == false && current_temp > desired_temp
+    status = "Fix the AC now! It's hot!"
+  elsif ac_working == false && current_temp < desired_temp
+    status = "Fix the AC whenever you have the chance. It's cool."
+  end
+  status
```

Comment:
Yep! In fact, because the string is the last thing evaluated each time, you could have even written 
``` 
if ac_working == true && current_temp > desired_temp
   "Turn on the AC."
...